src/Square.java

```diff
@@ -0,0 +1,50 @@
+public class Square {
+    /**
+     * The Square class models a Square on a Scrabble Board.
+     * Each square has the possibility to have a tile placed
+     * TODO Each square can have points
+     *
+     * @author Michael Kyrollos, 101183521
+     * @version 1.0
+     * @date October 22, 2022
+     */
+
+    private Tile tile;
+    public Square() {
+
```

Comment:
Consider initializing the only field in the Square class, which is 'tile', to null in the Square constructor.